packages/web-component-tester/browser/childrunner.ts

```diff
@@ -167,25 +194,34 @@ export default class ChildRunner {
   done() {
     util.debug('ChildRunner#done', this.url, arguments);
 
-    // make sure to clear that timeout
+    // Make sure to clear that timeout.
     this.ready();
     this.signalRunComplete();
 
-    if (!this.iframe)
-      return;
-    // Be safe and avoid potential browser crashes when logic attempts to
-    // interact with the removed iframe.
-    setTimeout(function() {
-      this.iframe.parentNode.removeChild(this.iframe);
-      this.iframe = null;
-    }.bind(this), 1);
+    if (this.iframe) {
+      // Be safe and avoid potential browser crashes when logic attempts to
+      // interact with the removed iframe.
+      setTimeout(() => {
+        const { iframe } = this;
+        iframe.removeEventListener(
```

Comment:
For this sort of cleanup work, I prefer an array of callbacks to call as a private member on the class. That way the cleanup work can be registered at the same time and in the same place as the stuff that needs cleanup.

That would also let us avoid the repetitive file-level constants for event names